src/gpodder/opml.py

```diff
@@ -90,6 +97,7 @@ def __init__(self, url):
                         outline.getAttribute('text')
                         or outline.getAttribute('xmlUrl')
                         or outline.getAttribute('url'),
+                    'section': section or 'audio',
```

Comment:
Should be "Audio" to match what gpodder already uses. It works great, however, importing an old OPML file places all channels in "audio". Without this patch, it checks the feed type and places them in "Audio" or "Video".

Reply:
Getting rid of ` or 'audio'` reverts to the old behavior when importing old feeds.

Reply:
thanks